WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Payments Menu/PaymentsRow.swift

```diff
@@ -4,11 +4,13 @@ struct PaymentsRow: View {
     private let image: Image
     private let title: String
     private let subtitle: String?
+    private let badgeImage: Bool
```

Comment:
Should we rename this to `hasBadgeImage`, `isBadgeVisible`, or similar to reflect the bool type?

Reply:
Yep, I'll go with `shouldBadgeImage`, but do it in a separate PR